bugbug/tools/code_review.py

```diff
@@ -1378,7 +1425,13 @@ def __init__(self, vector_db: VectorDB) -> None:
             model="text-embedding-3-large", api_key=get_secret("OPENAI_API_KEY")
         )
 
-    def clean_comment(self, comment):
+    def clean_comment(self, comment: str):
+        # We do not want to keep the LLM note in the comment, it is not useful
+        # when using the comment as examples.
+        llm_note_index = comment.find("> This comment was generated automatically ")
+        if llm_note_index != -1:
+            comment = comment[:llm_note_index]
```

Comment:
maybe also `strip()` to remove the last newline before `> This comment ...`?

Reply:
We have that later in line 1438: https://github.com/mozilla/bugbug/blob/b92e68c50a196982a14f3f35630bdbd1fa52bd89/bugbug/tools/code_review.py#L1438